charts/kong-collectorapi/README.md

```diff
@@ -0,0 +1,159 @@
+# Kong-Collector
+
+[Kong-Collector](https://konghq.com/products/kong-enterprise/kong-immunity) is
+an application which enables the use of Kong Brain and Kong Immunity.
+
+Kong Brain and Kong Immunity are installed as add-ons on Kong Enterprise, using
+a Collector API and a Collector Plugin to communicate with Kong Enterprise.
+
+## Introduction
+
+This chart bootstraps a
+[Kong-Collector](https://docs.konghq.com/enterprise/latest/brain-immunity/install-configure/)
+deployment on a [Kubernetes](http://kubernetes.io) cluster using the
+[Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Kong Enterprise version 1.3.0.2+
+  [chart](https://github.com/helm/charts/tree/master/stable/kong)
+
+## Installing the Chart
+
+(If you already have a Kong Admin API, skip to Step 4. )
+
+To install the chart with the release name `my-release`:
+
+1. [Add Kong Enterprise license
+   secret](https://github.com/Kong/charts/tree/master/charts/kong#kong-enterprise)
+
+2. [Add Kong Enterprise registry
+   secret](https://github.com/Kong/charts/tree/master/charts/kong#kong-enterprise-docker-registry-access)
+
+3. Set up Kong Enterprise with postgresql, overriding postgres host and setting
+   a port for kong manager to use the Kong Admin API
+
+```console
+$ helm install my-kong stable/kong --version 0.36.1 -f kong-values.yaml --set env.admin_api_uri=$(minikube ip):32001
```

Comment:
While we take every step to not break anything across multiple versions of the chart, it sometimes does happen. I'd recommend to use a specific version of the chart here using `--version` flag.

Reply:
I agree `--version 0.36.1`was the last version I tested this with